content/learn/quickstart.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Patterns quick start
+menu: learn
+weight: 20
+---
+
+== Patterns quick start
+
+Each pattern can be deployed using the command line. The only requirement is to have 
+`git` and `podman` installed. See the <<installation_prerequisites>> for more information.
+
+Check the `values-\*.yaml` for changes that are needed before deployment. After
+changing the `values-*.yaml` files where needed and pushing them to your git
+repository, you can run `./pattern.sh make install` from your local repository directory and
+that will deploy the datacenter/hub cluster for a pattern. Edge clusters are
+deployed by joining/importing them into ACM on the hub.
```

Comment:
Global comment: Do not hard wrap lines at 80 characters (or at any other length).

Check your editor settings that let you undo hard wrapping on a paragraph.

Reply:
Ack, noted!